### DISCUSSION TITLE
Is there an option to skip the select query in a create operation?

### DISCUSSION THREAD

**Discussion by stefanoruth** [prisma/prisma#3827] [Q&A] [answered]
I run a audit log in the database and are only inserting queries, i have no use for the inserted data at this point so i am trying to have the create function just return void, so it skips the part of the transaction where it fetches the data again from the database, anybody know if this is possible?

**Comment by ryands17** [comment, marked as answer]
Currently it's required to at least select 1 field. I would suggest opening a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for executing without select.

As an alternative, you can use [raw queries](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/raw-database-access) as a workaround.

**Reply by stefanoruth** [reply]
Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
